validator/libs/schema.js

```diff
@@ -69,6 +69,7 @@ module.exports = {
     const file = path.join(fullPath, fileSchema);
     const schema = openFile(file, "schema");
     const ajv = new Ajv(conf.ajvOptions);
+    ajv.addFormat("date-optionally-yearless", /^(\d\d\d\d|-)-[0-1]\d-[0-3]\d$/);
```

Comment:
please provide a more robust Reg Exp

Reply:
Ok, do yo mean something like this (removing support for dates using `/` and `. ` for separator): https://stackoverflow.com/questions/15491894/regex-to-validate-date-format-dd-mm-yyyy/15504877#15504877?

Reply:
the format should be `yyyy-mm-dd`  I would be fine if you are able to avoid 35 as day number :) and 13 as month number. Leap years etc. it is not needed to check ... I believe.